client/components/foldable-card/index.jsx

```diff
@@ -86,9 +87,11 @@ var FoldableCard = React.createClass( {
 		}
 		if ( this.props.children ) {
 			let iconSize = 24;
+			let screenReaderDefaultText = ( typeof this.translate === 'function' ) ? this.translate( 'More' ) : '';
```

Comment:
@jkudish fixed in https://github.com/Automattic/wp-calypso/pull/4297/commits/868dd9aedc8799acc3c77c8bb803cdd4072557b8